xml.go

```diff
@@ -21,11 +21,13 @@ func ConvertXML(r io.Reader) (string, map[string]string, error) {
 	return result, meta, nil
 }
 
+const maxBytes = 20 << 20 // 20MB
```

Comment:
Note: this is arbitrary

Reply:
nit: perhaps move this to limit.go. It seems specific to XML things but it's not so we should make that a bit clearer.